### PR TITLE
Added more details for full example.

### PR DESCRIPTION
**Pull request by sentakuhm**
added how to run hypridle and more details in full example.

### CODE REVIEW

File: pages/Hypr Ecosystem/hypridle.md [hunk 1]
```diff
@@ -30,31 +32,50 @@ Example listener:
...
+general {
+    lock_cmd = pidof hyprlock || hyprlock       # avoid starting multiple hyprlock instances.
+    before_sleep_cmd = loginctl lock-session    # lock before suspend.
+    after_sleep_cmd = hyprctl dispatch dpms on  # to avoid having to press a key twice to turn on the display.
+}
+
+# set monitor backlight to minimum
+listener {
+    timeout = 150                                # 2.5min.
+    on-timeout = brightnessctl -s set 0          # monitor backlight off.
```
Comment (vaxerski) [inline pages/Hypr Ecosystem/hypridle.md:55]: I wouldn't set this to 0 as on OLEDs that will actually mean literally 0

Comment (sentakuhm) [reply pages/Hypr Ecosystem/hypridle.md:55]: so, OLEDs users can escape monitor backlight listener?

Comment (vaxerski) [reply pages/Hypr Ecosystem/hypridle.md:55]: no, I'd just set it to like 10% instead

**Comment by sentakuhm** [comment]
@vaxerski done.

File: pages/Hypr Ecosystem/hypridle.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ hypridle is hyprland's idle management daemon.
 
 Configuration is done via the config file at `~/.config/hypr/hypridle.conf`.
 A config file is required; hypridle won't run without one.
+To run hypridle edit `hyprland.conf` and add:
```
Comment (vaxerski) [inline pages/Hypr Ecosystem/hypridle.md:9]: _to run hypridle at startup_

**Review by vaxerski — APPROVED** [review]
rest lgtm

File: pages/Hypr Ecosystem/hypridle.md (hunk 2 above)
Comment (sentakuhm) [reply pages/Hypr Ecosystem/hypridle.md]: done!.

**Pull request merged**